docs/providers/twitter.md

```diff
@@ -21,6 +21,10 @@ You can override any of the options to suit your own use case.
 
 ## Example
 
+:::note
+API Key == Consumer Key. API Secret Key == Consumer Secret. See `Terminology clarification` section under https://developer.twitter.com/en/docs/authentication/oauth-1-0a/obtaining-user-access-tokens
+:::
```

Comment:
Again, these are just environment variable names. No one cares/should care what they are. 🙂 I usually call it `TWITTER_ID` and `TWITTER_SECRET` because I don't like typing... The important part is to explain what the **values** should be. For that, I think we should explain what `TWITTER_CLIENT_ID` and `TWITTER_CLIENT_SECRET` **refer to** because that is what's below the note. You are closer with your new suggestion, but I would write something like this:

```suggestion
:::note
`TWITTER_CLIENT_ID` refers to the *consumer key*, while `TWITTER_CLIENT_SECRET` is the *consumer key secret*. Read more about the Twitter [terminology here](https://developer.twitter.com/en/docs/authentication/oauth-1-0a/obtaining-user-access-tokens#:~:text=Terminology%20clarification).
:::